tests/Block/Breadcrumb/BreadcrumbTest.php

```diff
@@ -23,7 +23,7 @@
  */
 class BreadcrumbTest extends AbstractBlockServiceTestCase
 {
-    public function testBlockService()
+    public function testBlockService(): void
```

Comment:
What if you extend this test class and don’t have the void return type when overriding this method? Is it ok?

Reply:
It's a test